app/views/shared/customer_support/_live_chat_widget.html.erb

```diff
@@ -40,6 +40,12 @@
     gap: 8px;
   }
 
+  /* Footer Text And Icon */
+  .cx-widget.cx-theme-dark .cx-branding *, .cx-widget.cx-theme-dark .cx-footer * {
```

Comment:
very likely is totally correct, but just want to confirm the use of the asterisk with 2 of the selectors here - i believe `.class *` would select any/all children under `.class`. Mostly my concern would be someone down the line feeling like they need to use an `!important` on a child element to override this.

If this makes sense for the use case, then ignore me!